src/binding.c

```diff
@@ -107,3 +117,13 @@ int bindOutput()
     fprintf(stdout, "info: successfully created virtual output device\n");
     return EXIT_SUCCESS;
 }
+
+/**
+ * Releases the virtual output device.
+ * */
+int release_output()

```

Comment:
Should this write a message to stdout like bind_output()?

Reply:
I would like to figure out how to find the virtual device information (like event path) first.

Reply:
Added